checkov/terraform/checks/resource/tencentcloud/VPCFlowLogConfigEnable.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import \
+    BaseResourceCheck
+
+
+class VPCFlowLogConfigEnable(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure VPC flow log disabled"
```

Comment:
```suggestion
        name = "Ensure Tencent Cloud VPC flow logs are disabled"
```

Reply:
Suggested title change, but why would you want to make sure Flow Logs are disabled? There is value in having these logs for security inspection.

Reply:
The title `Ensure VPC flow log disabled` description is incorrect. Actually, it should be `Ensure Tencent Cloud VPC flow logs are enable`

Reply:
That works great. Can you update the title to that?